docs/sources/setup/token.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Configure Token'
+menuTitle: Configure Token
+description: Configuring the GitHub personal access token
+keywords:
+  - data source
+  - github
+  - github repository
+  - API
+labels:
+  products:
+    - oss
+    - enterprise
+    - cloud
+weight: 102
+---
+
+# Configure Token
+
+You will need a **personal access token** to use the plugin. GitHub currently supports two types of personal access tokens:
```

Comment:
Don't use bold for emphasis, instead use italics: https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold

```suggestion
You will need a _personal access token_ to use the plugin. GitHub currently supports two types of personal access tokens:
```